menas/ui/service/MappingTableService.js

```diff
@@ -170,8 +170,11 @@ var MappingTableService = new function () {
       if (xhr.status === 400) {
         let err = "Disabling mapping table failed. Clear the following dependencies first:\n";
         let oData = JSON.parse(xhr.responseText);
-        for(let ind in oData) {
-          err += "\t - " + oData[ind].name + " (v. " + oData[ind].version + ")";
+
+        err += "Datasets:\n";
+        let datasets = oData["datasets"];
+        for(let ind in datasets) {
+          err += "- " + datasets[ind].name + " (v. " + datasets[ind].version + ")";
```

Comment:
Please add new lines and maybe some tabs. This is not readable enough. Maybe organize them in a table?

This counts for others additions in this PR too.

<img width="846" alt="screenshot 2019-03-05 at 17 58 37" src="https://user-images.githubusercontent.com/17467768/53822721-94645080-3f70-11e9-93d3-13a97b40a75d.png">

Reply:
That's spot on, I completely missed I hadn't added the new line. Thanks!
Not sure what you mean by `This counts for others additions in this PR too.`, though.

Reply:
There are 3 occurrences of the same error. Schemas, Datasets and MappingTables dependencies

Reply:
Ah, that was implicitly understood.

Reply:
No sure I understand what the error message is trying to tell me. Why disabling failed and what do I need to do to "clear" the dependencies. Consider rephrasing/simplifying the message text.

Also, the list of entities (datasets and versions I presume) seems quite long. Do we need to list everything?

Reply:
Good point, I think I'll reduce it to just the names of the dependencies and ask users to check the `Used In` tab for details (which will be formatted better in #213)